.github/workflows/label.yml

```
@@ -0,0 +1,19 @@
+---
+
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler/blob/master/README.md
+
+name: "Pull Request Labeler"
+on: [pull_request_target]
+
+jobs:
+  labeler:
```

```suggestion
  labeler:
    permissions:
      contents: read
      pull-requests: write
```

As the example of the action's docs.